lldb/packages/Python/lldbsuite/test/decorators.py

```diff
@@ -886,11 +885,14 @@ def skipIfAsan(func):
     """Skip this test if the environment is set up to run LLDB *itself* under ASAN."""
     return skipTestIfFn(is_running_under_asan)(func)
 
-def _get_bool_config_skip_if_decorator(key):
+def _get_bool_config(key):
     config = lldb.SBDebugger.GetBuildConfiguration()
     value_node = config.GetValueForKey(key)
     fail_value = True # More likely to notice if something goes wrong
-    have = value_node.GetValueForKey("value").GetBooleanValue(fail_value)
+    return value_node.GetValueForKey("value").GetBooleanValue(fail_value)
+
+def _get_bool_connfig_skip_if_decorator(key):
```

Comment:
```suggestion
def _get_bool_config_skip_if_decorator(key):
```

Reply:
The question is now: How did this still do the right thing?

Reply:
it didn't work for me, I only noticed this because of python exceptions

Reply:
Interesting, because the tests on swift CI actually did run as expected: https://ci.swift.org/job/apple-llvm-project-pr-linux/1530/console